dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnector.java

```diff
@@ -100,8 +99,11 @@ String getSql(@Nonnull Predicate<? super String> predicate) {
                     + "AND ST.CollectTimeStamp < CAST('%s' AS TIMESTAMP)\n",
                     SQL_FORMAT.format(interval.getStart()), SQL_FORMAT.format(interval.getEndExclusive())));
 
-            if (!Strings.isNullOrEmpty(condition))
-                buf.append(" AND ").append(condition);
+            if (!conditions.isEmpty()) {
```

Comment:
This `if` is redundant and can be removed. Similarly below.

Reply:
done